diff --git a/Dockerfile b/Dockerfile
index e764fdf..f502964 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -32,7 +32,10 @@ COPY geteduroam.conf /etc/apache2/sites-available/geteduroam.conf
 RUN a2ensite geteduroam
 
 RUN git clone https://github.com/geteduroam/letswifi-portal.git
+COPY enable-sunet-css.patch /letswifi-portal
+COPY sunet.css /letswifi-portal/www/assets/
 WORKDIR /letswifi-portal
+RUN git apply enable-sunet-css.patch
 RUN composer install && make SIMPLESAMLPHP_VERSION=2.2.2 simplesamlphp
 
 WORKDIR /var/www/
